sdk/cloudmachine/Azure.Projects.Provisioning/tests/Data/ofx.bicep

```diff
@@ -39,10 +39,10 @@ resource cm_storage_1_StorageBlobDataContributor 'Microsoft.Authorization/roleAs
   scope: cm_storage
 }
 
-resource cm_storage_cm_identity_StorageBlobDataContributor 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
-  name: guid('cm_storage', cm_identity.id, subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'ba92f5b4-2d11-453d-a403-e96b0029c9fe'))
+resource cm_storage_project_identity_StorageBlobDataContributor 'Microsoft.Authorization/roleAssignments@2022-04-01' = {
```

Comment:
Should we replace the leading "cm" also?

Reply:
same for the others below.

Reply:
I removed all where we spell out CloudMachine. I did not yet remove cm. Since we don't have great tests, I did not want to risk more debugging than necessary; need to prepare demo for next Thu :-)